### ISSUE TITLE
Readme says nothing about the project

### ISSUE THREAD

**Issue by buffet** [iiPlasma/sequestrum#34]
When you look at the readme without having checked on your dotfiles before, sequestrum says *nothing* about itself apart from "it's better than".

**Comment by iiPlasma** [comment]
I'm working on rewriting it atm

**Comment by Lilja** [comment]
Also needs requirements to build.

**Comment by Lilja** [comment]
@buffet Python version is a build/runtime requirement.

**Comment by iiPlasma** [comment]
@Lilja Good point, thanks for pointing that out! And @buffet pyyaml doesn't have to be a thing cause it auto installs if you don't have it
